tests/kernel/wave/wave_attention_test.py

```diff
@@ -196,7 +198,7 @@ def repeat(
         generate_iree_ref(
             "chain_mmt", [q, k, v], [iree_ref], config, run_bench=run_bench
         )
-        assert_close(output, iree_ref)
+        assert_close(output.cpu(), iree_ref)
```

Comment:
Instead of moving the output to the cpu, should we do check_device=False, which automatically moves the output to the cpu? https://pytorch.org/docs/main/testing.html